README.md

```diff
@@ -1,53 +1,57 @@
 
-### LRCubeRL
-Learning Rubik's Cube with Reinforcement Learning
-* [Code](https://github.com/riveSunder/lrcuberl)
+### OpenAI Safety Gym
+* [Code](https://github.com/openai/safety-gym) (MIT License) (MuJoCo dependency)
+* [Baselines](https://github.com/openai/safety-starter-agents)
+* [Blog](https://openai.com/blog/safety-gym/)
+* [Paper (pdf)](https://cdn.openai.com/safexp-short.pdf)
 
-### CARLE
-Cellular Automata reinforcement learning environment
-* [Code](https://github.com/rivesunder/carle)
-* [Blog post: How to break your random network distillation](https://rivesunder.gitlab.io/rl/2019/08/24/breaking_rnd.html)
 
-### RL Simple Games
-Real simple games, like tic-tac-toe and hexapawn.
-* [Code](https://github.com/riveSunder/rl-simple-games)
+### SafeLife
+* [Code](https://github.com/PartnershipOnAI/safelife) (Apache 2.0)
+* [Blog (Partnership on AI)](https://www.partnershiponai.org/safelife/)
+* [Blog (Weights and Biases)](https://wandb.ai/stacey/saferlife/reports/Measuring-Safety-in-Reinforcement-Learning--VmlldzoyNjk3MTM)
+* [Benchmarks](https://wandb.ai/safelife/v1dot2/benchmark)
+* [Paper](https://arxiv.org/abs/1912.01217)
 
-* Notes: This is a collection of low-complexity games built as RL environments. The environments are part of a nascent project to build a [game agnostic game learner](https://github.com/riveSunder/game-agnostic)
 
-### Open Safety Gym
-Coming Soon!
- 
-# Cool and Little-Known Environments
+### AI safety gridworlds
+* [Code](https://github.com/deepmind/ai-safety-gridworlds) (Apache 2.0)
+* [Paper](https://arxiv.org/abs/1711.09883)
+* 
 
-### balance-bot by yconst
-* [Code](https://github.com/yconst/balance-bot)
-* Blog [part I](https://backyardrobotics.eu/2017/11/27/build-a-balancing-bot-with-openai-gym-pt-i-setting-up/), [Part II](https://backyardrobotics.eu/2017/11/29/build-a-balancing-bot-with-openai-gym-pt-ii-the-robot-and-environment/)
-* Notes: This environment is part of a tutorial on building an RL environment using the open source [PyBullet](https://pybullet.org) physics engine. The task is a self-balancing robot that must learn to stay upright. I haven't experimented with the environment yet, but the tutorial on building robots with PyBullet is really useful. If that sounds interesting you'll also want to dive into the [PyBullet quickstart guide](https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit)
+<!-- ### Safer RL
```

Comment:
Go ahead and uncomment this link, but leave the note about the state of the project.